src/modules/shared/pages/notifications/notifications-list.component.html

```diff
@@ -103,12 +103,13 @@ <h2 class="nhsuk-heading-xs nhsuk-u-margin-0">Filters</h2>
                   </a>
                 </ng-container>
 
-                {{ 'shared.catalog.innovation.notification_context_details.' + item.contextDetail + '.title' | translate: item.params }} 
-
-                <a *ngIf="item.link" href="/transactional{{ item.link.url }}" (click)="onNotificationClick($event, item.id, item.link.url, item.link.queryParams)">
-                  {{ 'shared.catalog.innovation.notification_context_details.' + item.contextDetail + '.link' | translate: item.params }}
-                  <span class="nhsuk-u-visually-hidden"> {{ item.link.label }} </span>
-                </a>
+                <ng-container *ngIf="item.link">
```

Comment:
What if the notification doesn't have a link?

Reply:
Lines 95-104 take care of that part. 
What was missing was wrapping the whole 106-111 on a container for when it DOES have a link. Only the <a> component was conditional, so the title on 106 was always displaying.

Reply:
Makes sense! Sorry, I hadn't seen that.